keyboards/handwired/dactyl_manuform/4x5/keymaps/kombo/keymap.c

```diff
@@ -0,0 +1,211 @@
+/* auth: domonkos */
+
+#include "dactyl_manuform.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```